client/components/Info Panel/InfoPanel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { RiFolderUploadFill } from 'react-icons/ri';
+
+import './InfoPanel.css';
+
+const InfoPanel = () => {
+  const uploadExcelHandler = (e) => {
+    e.preventDefault();
+    console.log(`File was sent!`);
```

Comment:
A TODO here (for future iterations) maybe is to have a text that would be displayed on front end (and then hidden afterwards) instead of console logging.

Also, we may also want to alert the user if the upload fails.

Reply:
Oh for sure! Adding that on the task list